### PR TITLE
Add close button to fullscreen conversation dialog (on mobile)

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Quick fix : display a close button on mobile

CleanShot 2023-01-09 at 15 58 30

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts [hunk 1]
```diff
@@ -82,7 +83,17 @@ export class HaVoiceCommandDialog extends LitElement {
       return html``;
     }
     return html`
-      <ha-dialog open @closed=${this.closeDialog}>
+      <ha-dialog open @closed=${this.closeDialog} .heading=${"aaa"}>
```
Comment (balloob) [inline src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts:86]: aaa?

File: src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts [hunk 2]
```diff
@@ -82,7 +83,17 @@ export class HaVoiceCommandDialog extends LitElement {
...
+        <div slot="heading" class="heading">
```
Comment (balloob) [inline src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts:87]: any way we could only show this when the dialog is in full-screen?

Comment (balloob) [reply src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts:87]: oh nm you already do 👍

**Review by balloob — DISMISSED** [review]

File: src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts (hunk 1 above)
Comment (piitaya) [reply src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts]: Oops, I removed it !

**Pull request merged**